var/spack/repos/builtin/packages/py-pyarrow/package.py

```diff
@@ -30,17 +31,22 @@ class PyPyarrow(PythonPackage, CudaPackage):
     depends_on('pkgconfig', type='build')
     depends_on('python@3.5:', type=('build', 'run'), when='@0.17:')
     depends_on('python@3.6:', type=('build', 'run'), when='@3.0.0:')
+    depends_on('python@3.7:', type=('build', 'run'), when='@7.0.0:')
     depends_on('py-setuptools', type='build')
+    depends_on('py-setuptools@38.6.0:', type='build', when='@7.0.0:')
     depends_on('py-setuptools-scm', type='build', when='@0.15.0:')
     depends_on('py-cython', type='build')
     depends_on('py-cython@0.29:', type='build', when='@0.15.0:')
+    depends_on('py-cython@0.29.22:', type='build', when='@7.0.0:')
```

Comment:
Where do you see this requirement?

Reply:
I was looking at the latest version in PyPi which I thought was version 7.0.0, but 8.0.0 was released some days ago. So I was looking at the wrong requirements..